t/include.t

```diff
@@ -0,0 +1,24 @@
+use Test::Most;
+use Test::Lib;
```

Comment:
We need to make sure if `Test::Lib` is available on our usual targets as a package (otherwise we need to package it for those distros which are missing it).

If there's an easy way to test without `Test::Lib`, it might worth considering to replace it with either a perl core module or equivalent.